src/server/utils/serverConstants.js

```diff
@@ -0,0 +1,4 @@
+import ms from 'ms';
```

Comment:
Interesting! There are probably some other thing from .env.example we could pull in here such as `INVITE_TOKEN_KEY_HASH_ROUNDS`. Next time!